doc/developers/contributing.rst

```diff
@@ -383,6 +383,17 @@ documentation with the maths makes it more friendly towards
 users that are just interested in what the feature will do, as
 opposed to how it works "under the hood".
 
+You may also be asked to show your changes when it's built. When you create
+a pull request or make changes in an existing one modifying the docs, CircleCI
+automatically builds them. Thus, you can easily view your changes in the built
+artifacts using the following formula:
+
+``http://scikit-learn.org/circle?{BUILD_NUMBER}``
+
+Note: When you visit the details page of the CircleCI tests, you can find your
+BUILD_NUMBER mentioned as 'build #' which is different from your pull request number
```

Comment:
I was using this as my reference for both PR as well as Build #. Thought this has both of them clearly mentioned. 

![screenshot from 2016-11-16 15 17 08](https://cloud.githubusercontent.com/assets/13185699/20342401/4276ae8a-ac10-11e6-98bd-df07b8415bc6.png)

Do you feel I should be changing it then? Because mentioning both of them makes it a bit messy, I suppose.